.eslintrc.js

```diff
@@ -1,27 +1,41 @@
 module.exports = {
```

Comment:
we have lines like `{ allowSingleLine: true }` and `{commonjs: true, amd: true}`. Can we use same style space-in-brackets style?

Reply:
I will set `space-in-brackets` to always because it has less errors (550 vs 2400)